### DISCUSSION TITLE
Wunderground on Hassos 9.5

### DISCUSSION THREAD

**Discussion by Boomerang51** [cytech/Home-Assistant-wundergroundpws#157] [Q&A]
Good morning.
I am a new user, (inexperienced), I would like to ask for help.
I installed Hassos 9.5 on a virtual machine (VirtualBox), everything works fine. Now I would like to connect my weather station, (recorded on Wunderground) to Home Assistant.
I followed the instructions, downloaded the v1.xx file, and exploded it to an external folder.
The problem arises when I have to copy the contents of the www directory from the extracted file into an existing www directory in the .homeassistant directory. I can NOT find the .homeassistant dierctory. The only directories I see under "config" which I assume on Hassos, corresponds to Homeassistant, are: blueprints, custom_components, packages and tts.
I would be very grateful to anyone who could help this rookie, inexperienced. Thank you

**Comment by cytech** [comment]
good morning
you can just copy the www directory from the extracted file to the config directory.
instructions are a bit confusing because some installations already have a www directory.

**Comment by Boomerang51** [comment]
Thanks for the prompt reply.
I removed all the old v1.0.0 configuration and copied the new wundergroundpws v2.00, in the custom_components dir, and after restarting, I went to home assistant, selected devices and service, and under integrations, I can't find wundergroundpws.
Shouldn't I put ANYTHING in configuration.yaml ?
sorry for these elementary questions. Thank you

**Comment by cytech** [comment]
v2.x.x is still under development, not released...
with that said, with v2.x.x there should not be anything referring to wundergroundpws in configuration.yaml.
if you want to experiment with v2.x.x, feel free to do so at your own risk.
it seems pretty stable at this point.
if you have downloaded the zip file for the v2.x.x branch amd followed the install/upgrade instructions for that branch, wundergroundpws should show up in the add integration dialog.

**Comment by Boomerang51** [comment]
Thank you for the information.
In fact I tried to install the 2.0 release since I was unable to get a positive result with the previous one (surely due to my inexperience). Anyway I followed all the procedure you indicated, but unfortunately wundergroundpws doesn't appear in the add integration window. I'll make a few more attempts. I'll send you updates on what I'm doing. Thanks again and good job.

**Comment by Boomerang51** [comment]
Hi, I made several attempts to install the program that connects to wunderground and sends data to the home assistant, unfortunately, I failed, not even with the v1.1.1 version.
I was wondering if eventually, there might be the possibility to connect to my notebook with the TeamWiever program since I mounted "Hassos" on the Virtualbox virtual machine.
So you could do the tests with my station.
If you don't mind, I'm available for any test.
Thanks a greeting from Gianni

**Comment by Boomerang51** [comment]
Good morning.
I finally managed to install the wunderground interface. I used an indirect method (detection of the entities, and subsequent addition to the dashboard). Thanks for everything.
I am waiting for the new release.
Good work

**Comment by cytech** [comment]
:ok_hand:

**Comment by cytech** [comment]
closed as outdated

**Comment by cytech** [comment]
closed as outdated
